nautobot_chatops/dispatchers/mattermost.py

```diff
@@ -461,7 +472,13 @@ def prompt_for_text(self, action_id, help_text, label, title="Your attention ple
         }
         blocks = [textentry]
         # In Mattermost, a textentry element can ONLY be sent in a modal Interactive dialog
-        return self.send_blocks(blocks, callback_id=action_id, ephemeral=False, modal=True, title=title)
+        return self.send_blocks(
+            blocks,
+            callback_id=action_id,
+            ephemeral=settings.PLUGINS_CONFIG["nautobot_chatops"]["send_all_messages_private"],
```

Comment:
Per the Mattermost comment just above, I'd think that this one needs to stay `ephemeral=False` - but maybe I'm mistaken? Definitely would want to see this tested with Mattermost.

Reply:
Reverted, for the same reason below.